logistics_project/apps/malawi/tests/product.py

```diff
@@ -4,15 +4,17 @@
 from logistics_project.apps.malawi.tests.util import create_hsa
 from logistics_project.apps.malawi.tests.base import MalawiTestBase
 from django.conf import settings
+from static.malawi.config import BaseLevel, SupplyPointCodes
+
 
 class TestProductLevels(MalawiTestBase):
     
-    def testEmergencyLevels(self):
+    def testStaticLoader(self):
         csv_file = open(settings.STATIC_PRODUCTS, 'r')
         try:
             count = 0
-            expected_count = Product.objects.exclude(emergency_order_level=None).count()
-            static_sp = SupplyPoint.objects.all()[0]
+            # EPI Products aren't loaded through the static loader so exclude them here
```

Comment:
makes sense. figured it was something like that and not sure it should be a priority to fix. I think what probably happened is they started adding products on the site and we never back-ported those changes.